templates/taxi/car_list.html

```diff
@@ -1,17 +1,31 @@
-{% extends "base.html" %}
+{{% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
-  {% if car_list %}
-    <ul>
-      {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
-      {% endfor %}
-    </ul>
-  {% else %}
-    <p>There are no cars in taxi</p>
-  {% endif %}
+<h1>Car list<a style="float: initial" href="{% url 'taxi:car-create' %}"> +</a></h1>
```

Comment:
According to the checklist, there should be a `Cancel` button apart from the `Delete` one. The `Cancel` button will lead to the previous page the user was on.